src/test/java/seedu/address/model/task/DateTimeDueTest.java

```diff
@@ -0,0 +1,54 @@
+package seedu.address.model.task;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.testutil.Assert.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+public class DateTimeDueTest {
+
+    @Test
+    public void constructor_null_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> new DateTimeDue(null));
+    }
+
+    @Test
+    public void constructor_emptyString_throwsIllegalArgumentException() {
+        String emptyString = "";
+        assertThrows(IllegalArgumentException.class, () -> new DateTimeDue(emptyString));
+    }
+
+    @Test
+    public void constructor_invalidDateTime_throwsIllegalArgumentException() {
+        String invalidDateTime = "aaa";
+        assertThrows(IllegalArgumentException.class, () -> new DateTimeDue(invalidDateTime));
+    }
+
+    @Test
+    public void isValidDateTimeDue() {
+        // null email
```

Comment:
i think this should be null dateTime instead of email?

Reply:
Yes, right you are! I will change this.